docs/middleware.md

```diff
@@ -0,0 +1,64 @@
+# Middleware
+
```

Comment:
Can we please state explicitly that this solution for middleware is not explicitly supported and is subject to change?

> Note: Using Express Middleware is not explicitly supported and is subject to change in future releases of the framework.